src/features/Calendar/Calendar.js

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react';
+import moment from 'moment';
+import {useSelector, shallowEqual, useDispatch} from "react-redux";
+import {setPickedDate, getCalendarInitialData, daysOfWeek} from "./calendarSlice";
+import './Calendar.scss';
+
+const Calendar = () => {
+  const dispatch = useDispatch();
+  const [today] = useState(moment().format('YYYY-MM-DD'));
+  const {calendarStructure, pickedDate, pickedMonth, monthData} = useSelector((state) => {
+      return {
+        calendarStructure: state.calendar.calendarStructure,
+        pickedDate: state.calendar.pickedDate,
+        pickedMonth: state.calendar.pickedMonth,
+        monthData: state.calendar.monthData,
+      }
+    },
+    shallowEqual
+  );
+
+  useEffect(()=> {
+    dispatch(getCalendarInitialData(today));
+    dispatch(daysOfWeek());
+
+  }, [today, dispatch]);
+
+  const getDay = (value) => moment(value).format('DD');
+  const cutWeekDay = (value) => value.substring(0, 3);
+
+  const getMonthData = () => {
+    if (calendarStructure.length) {
+      const formattedMonth = moment(pickedMonth).startOf('month').format('YYYY-MM-DD');
+      return calendarStructure.find((item) => item.month === formattedMonth).dates;
+    }
+    return [];
+  };
+
+  const datePickHandle = (day, dayIndex) => {
+    console.log('Yo kliknales w dzien: ', day);
```

Comment:
ten consol log to tu na  stałe czy do testów ?

Reply:
testy